packages/client/components/TopBarNotifications.tsx

```diff
@@ -44,10 +45,21 @@ const TopBarNotifications = ({queryRef}: Props) => {
   const {edges} = notifications
   const hasNotifications = edges.some(({node}) => node.status === 'UNREAD')
   const menuContentRef = useRef<HTMLDivElement>(null)
-  const {togglePortal, originRef, menuPortal, menuProps} = useMenu<HTMLDivElement>(
+  const {togglePortal, openPortal, originRef, menuPortal, menuProps} = useMenu<HTMLDivElement>(
     MenuPosition.UPPER_RIGHT,
     {menuContentRef}
   )
+  const {location} = useRouter()
+  useEffect(() => {
+    const parsed = new URLSearchParams(location.search)
+    if (parsed.get('openNotifs')) {
+      // :HACK: Opening the portal immediately sometimes leads to rendering just off the left side
+      // of the window, so delay a bit before opening. This might be benefial regardless, as it
+      // catches the user's attention when it opens shortly after the rest of the page loads.
+      setTimeout(openPortal, 500)
+    }
+  }, [])
```

Comment:
+1 do we want to include `location` in `useEffect` deps?

Reply:
Hmmm I'm actually not sure. I _think_ no, since we only want this to have an effect on initial page load.  After that, the page should function like normal.